src/services/push.js

```diff
@@ -0,0 +1,221 @@
+var request = require('got');
```

Comment:
using 'got', a dependency that was being use called 'request' is not maintained anymore. Also got supports promises and async await nicely.